recipes/yaehmop/build.sh

```diff
@@ -0,0 +1,5 @@
+cd $SRC_DIR/tightbind
+make
+
+mv bind yaehmop
+cp yaehmop $PREFIX
```

Comment:
Executables should be copied to `$PREFIX/bin`. Is this an executable?

Reply:
Yep, fixed